### DISCUSSION TITLE
Suggestions on increasing pen types

### DISCUSSION THREAD

**Discussion by lhbadsl** [xournalpp/xournalpp#2697] [Ideas]
As a note-taking software, the most important function of xournal++ is that you can use the tablet to record. As a designer and linux fan, I often use it when watching tutorial videos, meetings, or when inspiration comes suddenly. However, I think it can become more professional, and the most important thing is to increase the types of calligraphy pens, because the user's calligraphy ability can make notes more perfect. I am tired of IPAD and WIN10, but the only regret is that there is no calligraphy application in LINUX, which I have to say is a pity.

**Comment by rolandlo** [comment]
We are making one step into that direction in #2166. Feel free to provide some details of what types of calligraphy pens you would find the most desirable.

**Reply by lhbadsl** [reply]
I have often used INKredible to write, but since I started using the LINUX system, I can only use the app on my Android tablet. There is a pen tool in it. The default pen in this app, I can’t describe its characteristics. But I can use them to write beautiful hard pen characters, just like with a real pen. You can download the app on Android to experience it and you will know. In addition, I use zenbrush3 to write the writing brush characters. This application can simulate the real brush effect. I usually use it on the iPad PRO. Of course, the brush is not necessary for the note-taking application; I hope to see the pen in XOURNAL++. Make the writing effect pretty good, just like it is in INKredible.

**Reply by lhbadsl** [reply]
Of course, I can also use pressure sensitivity to write effects similar to real ballpoint pens in Xournal++, using highlighter markers, but LINUX lacks applications that can write pen and brush effects, making it difficult for calligraphy enthusiasts to use their expertise in LINUX systems. KRITA can provide top-notch painting experience, but it is not born for calligraphy, and the writing experience is not ideal, while XOURNAL++ is born for writing, and the writing experience is very good, even though it only has two pen tools.

**Reply by lhbadsl** [reply]
#2166 is werry good。
